app/src/pages/inside/stepPage/modals/makeDecisionModal/elements/itemHeader/itemHeader.scss

```diff
@@ -65,6 +65,7 @@
   text-decoration: none;
   white-space: nowrap;
   overflow: hidden;
+  max-width: 90%;
```

Comment:
Please recheck this solution for screen 768px and more. 
Also does it expected behavior for screen size 768-990px
![image](https://user-images.githubusercontent.com/42434528/142475667-5c3639c7-59bd-4123-9ad1-9b051858d1df.png)